thrift/zipkincore.thrift

```diff
@@ -164,21 +164,21 @@ struct Endpoint {
    *
    * Ex for the ip 1.2.3.4, it would be (1 << 24) | (2 << 16) | (3 << 8) | 4
    */
-  1: i32 ipv4
+  1: required i32 ipv4
```

Comment:
changing this has very far-reaching side effects across multiple languages. What are you trying to accomplish here?